examples/step-93/doc/results.dox

```diff
@@ -0,0 +1,188 @@
+<h1>Results</h1>
+
+When you run the program with the step target function (in 2D), the output looks something like this:
+
+@code
+Number of active cells: 16384
+Number of degrees of freedom: 66049+66049+66049 = 132102
+Number of nonlocal dofs: 4
+Beginning solve...
+Wall time: 68.4734s
+Solved in 39973 MINRES iterations.
```

Comment:
```suggestion
Solved in 39973 MINRES iterations.
```
Can we use a better preconditioner by default, even if it is a very simple one like ILU? I know it is mentioned in possibilities for extension, but students sometimes haphazardly write `PreconditionIdentity` in their own solvers because they saw it in a tutorial program.